client/my-sites/plans/ecommerce-trial/entrepreneur-plan/entrepreneur-plan.jsx

```diff
@@ -146,7 +146,7 @@ export function EntrepreneurPlan() {
 						<h3 className="entrepreneur-trial-plan__plan-title">{ translate( 'Entrepreneur' ) }</h3>
 						<p className="card-text">
 							{ translate(
-								"Continue enjoying the full benefits of Entrepreneur plan, simply add your payment method and maximize your store's potential."
+								"Secure the full benefits of the Entrepreneur Plan. Purchase today and maximize your store's potential!"
```

Comment:
Should plan names not remain lower-case for consistency with the rest of Calypso?

```suggestion
								"Secure the full benefits of the Entrepreneur plan. Purchase today and maximize your store's potential!"
```

Reply:
Sure, just updated it.